rnnalyse/models/language_model.py

```diff
@@ -3,23 +3,27 @@
 from overrides import overrides
 from torch import Tensor, nn
 
-from ..typedefs.models import FullActivationDict
+from ..typedefs.activations import FullActivationDict
 
 
 class LanguageModel(nn.Module):
     """ Abstract class for LM with intermediate activations """
-    def __init__(self) -> None:
-        super(LanguageModel, self).__init__()
+    def __init__(self,
```

Comment:
Did you opt for the underscore names here because this class is supposed to be used as an abstract class?

Reply:
Yes, although I can imagine this setup is not perfect yet. Do you have more experience with setting up abstract classes? I'm not sure how something like `abc` can be used here if we still want `LanguageModel` to inherit from `nn.Module`.

Reply:
Let me know what you think of my latest addition to this class, not super sure if multiple inheritance is the way to go here but my compiler isn't complaining :p

Reply:
I think this works,  but is it even necessary to inherit from ABC. The class is already abstract because it has some abstract methods that need to be implemented. Maybe just inheriting from LanguageModel is enough?

Reply:
I think it looks a bit strange because you use underscore variables and names for protected attributes and class methods. I would just inherit from ABC (abstract base class) from the `abc` package as well, in this class you also get an error if you try to instantiate an object from that class.